src/docs/user/hcatalog.txt

```diff
@@ -61,6 +61,10 @@ The presence of the +--hcatalog-table+ option signifies that the import
 or export job is done using HCatalog tables, and it is a required option for
 HCatalog jobs.
 
++--hcatalog-external-table+::
```

Comment:
Could you please extend the documentation of --drop-and-create-hcatalog-table option that it will have affect on external HCat tables too?

Reply:
You are correct, thanks for notifying me about the use case with delete. I think the provided example fits there.